sdk-playground/packages/client/src/discordSdk.tsx

```diff
@@ -1,32 +1,46 @@
-import {DiscordSDK, DiscordSDKMock, IDiscordSDK, patchUrlMappings} from '@discord/embedded-app-sdk';
+import {
+  DiscordSDK,
+  DiscordSDKMock,
+  IDiscordSDK,
+  patchUrlMappings,
+} from "@discord/embedded-app-sdk";
 
 const queryParams = new URLSearchParams(window.location.search);
-const isEmbedded = queryParams.get('frame_id') != null;
+const isEmbedded = queryParams.get("frame_id") != null;
 
 let discordSdk: IDiscordSDK;
 
 if (isEmbedded) {
-  discordSdk = new DiscordSDK(import.meta.env.VITE_CLIENT_ID, {disableConsoleLogOverride: true});
+  discordSdk = new DiscordSDK(import.meta.env.VITE_CLIENT_ID, {
+    disableConsoleLogOverride: true,
+  });
   patchUrlMappings([]);
 } else {
-  discordSdk = new DiscordSDKMock(import.meta.env.VITE_CLIENT_ID, null, null);
+  discordSdk = new DiscordSDKMock(
+    import.meta.env.VITE_CLIENT_ID,
+    null,
+    null,
+    null
```

Comment:
this is the only change, adding a `null` location id. Everything else in this file is due to auto-formatting